cluster/packages.sls

```diff
@@ -39,6 +39,18 @@ install_cluster_packages:
 
 {% endif %}
 
+{% if cluster.qdevice is defined %}
+{% if cluster.qdevice.qnetd_hostname is defined %}
```

Comment:
These conditions might be combined, but I see in future if there could be more parameters under `cluster.qdevice` So this is fine

Reply:
Thanks, yes, the qdevice has more arguments that should come in future commits.